src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Helpers.cs

```diff
@@ -242,6 +242,57 @@ public static ReadOnlyMemory<byte> DecodeOctetStringAsMemory(ReadOnlyMemory<byte
                 throw new CryptographicException(SR.Cryptography_Der_Invalid_Encoding, e);
             }
         }
+
+        public static bool AreSamePublicECParameters(ECParameters aParameters, ECParameters bParameters)
```

Comment:
This was pulled from `ECDsaCertificateExtensions` and changed to accept the `ECParameters`, and renamed to be clear this is only comparing the public properties of `ECParameters`.